lints/base.go

```diff
@@ -216,10 +221,24 @@ func (report *LintReport) Execute(cert *x509.Certificate) error {
 			return err
 		}
 		l.updateReport(report, res)
+		updateErrorStatePresent(report, res)
 	}
 	return nil
 }
 
+func updateErrorStatePresent(report *LintReport, result ResultStruct) {
```

Comment:
To fill out the protobuf, we need to have this data available. I'd rather not iterate through every lint just to determine that there's an error or warning present, it's better if ZLint itself keeps track of that stuff and just exposes a bool to the caller